release-notes/10.0/preview/preview1/wpf.md

```diff
@@ -2,7 +2,7 @@
 
 This release was focused on quality improvements to WPF. You can find detailed information about the improvements in the GitHub Release below:
 
-- [WPF GitHub Release Notes](https://github.com/dotnet/wpf/releases)
+- [WPF GitHub Release Notes](https://github.com/dotnet/wpf/releases/tag/v10.0.0-preview.1.25080.3)

```

Comment:
Was this change intentional?

Reply:
No. Something weird must have happened when the base branch changed.